doc/sjs-for-js/es6-to-scala-part3.md

```diff
@@ -260,6 +260,234 @@ Here we use triple-quoted strings that allow us to write regex without escaping
 converted into a {% scaladoc util.matching.Regex %} object with the `.r` method. Because regexes extract strings, we need
 to convert matched groups to integers ourselves.
 
+## Functions revisited
+
+We covered the basic use functions in [Part 1](es6-to-scala-part1.html), but Scala, being a functional programming
+language, provides much more when it comes to functions. Let's explore some of the more advanced features and how they
+compare to JavaScript.
+
+#### Higher-order functions
+
+Scala allows the definition of higher-order functions. These are functions that take other functions as parameters, or
```

Comment:
Scala<ins>, as JavaScript,</ins> allows [...]

As written, the sentence implies that JS doesn't allow that.